src/test/expression/expression_evaluator_to_values_test.cpp

```diff
@@ -272,6 +272,30 @@ TEST_F(ExpressionEvaluatorToValuesTest, PredicatesLiterals) {
   EXPECT_TRUE(test_expression<int32_t>(*between_(5.1f, null_(), 5), {0}));
   EXPECT_TRUE(test_expression<int32_t>(*between_(null_(), 3.1, 5), {std::nullopt}));
   EXPECT_TRUE(test_expression<int32_t>(*between_(null_(), null_(), null_()), {std::nullopt}));
+  EXPECT_TRUE(test_expression<int32_t>(*between_lower_exclusive_(3, 3.0, 5.0), {0}));
```

Comment:
(II )the tests you added would pass even if all between types were rewritten to the same expression. Please write tests that actually test the lower and upper bound.

Reply:
what about now?